ui/webui/resources/leo/web_components.ts

```diff
@@ -14,4 +14,13 @@ import iconsMeta from '@brave/leo/icons/meta'
 
 setIconBasePath('//resources/brave-icons')
 
+const variablesUrl = '//resources/brave/leo/css/variables.css'
+const variablesLink = document.querySelector(`link[rel=stylesheet][href$="${variablesUrl}"]`)
+if (!variablesLink) {
```

Comment:
Can we add a comment about when this can happen? I guess that's when the topmost ui is the upstream's one.